### PR TITLE
Feature/delete meetings

### PR DESCRIPTION
**Pull request by Asergey91**
not sure if I deleted everything

### CODE REVIEW

**Review by lagartoverde — APPROVED** [review]
Looks good :)

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-FROM semtech/mu-javascript-template
-LABEL maintainer="madnificent@gmail.com"
\ No newline at end of file
+FROM nvdk/mu-javascript-template
```
Comment (nvdk) [inline Dockerfile:1]: not sure if you need to extend my fork, you only need it if you added tests

File: app.js [hunk 2]
```diff
@@ -1,4 +1,64 @@
-import {app} from "mu";
-app.get('/', function( req, res ) {
-  res.send('hello world')
+import { app, update, query, sparqlEscapeUri, errorHandler } from 'mu';
+
+app.get('/', function (req, res) {
+  res.send('Hello mu-javascript-template');
```
Comment (nvdk) [inline app.js:4]: probably makes sense to change this to the correct service name

File: app.js [hunk 3]
```diff
@@ -1,4 +1,64 @@
...
 });
+
+app.delete('/:id', async function (req, res) {
+
+  const meetingId = "http://data.lblod.info/id/zittingen/" + req.params.id;
```
Comment (nvdk) [inline app.js:9]: I think it's safer (and more compliant with semanticworks) to look up the uri based on the mu:uuid. We might change the baseURI at some point...

File: app.js [hunk 4]
```diff
@@ -1,4 +1,64 @@
...
+
+  // Get all needed ids
```
Comment (nvdk) [inline app.js:11]: comment does not seem in line with what the query does

**Review by nvdk — COMMENTED** [review]
I think some changes are required

**Pull request merged**